### ISSUE TITLE
paradox-menu-view-commit-list says "Package nil is not a GitHub repo"

### ISSUE THREAD

**Issue by bk138** [Malabarba/paradox#176]
While I can jump to a package's github repo when pressing `v`, `l` does not work and says "Package nil is not a GitHub repo".

My config is here: https://github.com/bk138/dotfiles/blob/master/.emacs

**Comment by notuntoward** [comment]
I have the same problem, although this did work a couple of weeks ago, when I first installed paradox.

**Comment by blahgeek** [comment]
Run `paradox--refresh-remote-data` manually works for me

**Comment by bk138** [comment]
> Run `paradox--refresh-remote-data` manually works for me

Yeah true. Should be called by paradox itself and not the user, or? 

Also, seems like #179 will not fix this issue here.

**Comment by notuntoward** [comment]
Unfortunately, `M-x paradox--refresh-remote-data` doesn't fix this problem, at least not for me.  Below is the result I see after running `M-x paradox--refresh-remote-data`, putting the cursor on the org-roam package, and hitting `f l`.  

This problem occurs only for some packages, not all of them.

![image](https://user-images.githubusercontent.com/3851891/84395605-f0297a80-abb2-11ea-9b8d-9e0369b319ac.png)

**Comment by hmelman** [comment]
I assume this is the same as #167 and it's hitting GitHub rate limits

**Comment by notuntoward** [comment]
Paradox always misses the same packages, so it doesn't seem like a rate limit problem.

**Comment by hmelman** [comment]
> Paradox always misses the same packages, so it doesn't seem like a rate limit problem.

I agree.

**Comment by Knusper** [comment]
> Run `paradox--refresh-remote-data` manually works for me

this also fixes it for me...

**Comment by hmelman** [comment]
> Paradox always misses the same packages, so it doesn't seem like a rate limit problem.

I think the problem is that the data at https://raw.githubusercontent.com/Malabarba/paradox/data/data-hashtables hasn't been updated in many months.  So packages newer than that are not recognized by paradox.

**Comment by notuntoward** [comment]
For me, `paradox--refresh-remote-data` doesn't help.  For example, paradox still says that the org-roam package is not in GitHub.

org-roam also isn't in https://raw.githubusercontent.com/Malabarba/paradox/data/data-hashtables, supporting @hmelman's theory.

**Comment by younger-1** [comment]
The CI for updating data was down.

**Comment by Malabarba** [comment]
I'll be honest in that I haven't quite had the time to keep up with paradox maintenance. 
The data hashtables haven't been updated in a long time, and this isn't the first time it happens. 

If someone would like to configure a github workflow to run the scripts automatically that would be much appreciated. 
Alternatively, I've been considering just archiving this package as I no longer think it contributes enough over the regular package.el to be worth the trouble.

**Comment by hmelman** [comment]
> Alternatively, I've been considering just archiving this package as I no longer think it contributes enough over the regular package.el to be worth the trouble.

@Malabarba I think paradox-commit-list.el is great and would like to see it in package.el.
